modules/database_module.rb

```diff
@@ -0,0 +1,18 @@
+module Database
+  FOLDER_PATH = './database/'.freeze
+  FILE_PATH = 'test'.freeze
+  FORMAT_PATH = '.yml'.freeze
+  PATH = FOLDER_PATH + FILE_PATH + FORMAT_PATH
+
+  def load_db
+    entities = [Book, Reader, Author, Order, Time]
+    data = YAML.safe_load(File.open(PATH), entities, [], [], true)
+    data.each { |i| i.each { |y| add(y) } }
```

Comment:
you shouldn't use method `add` here. It's method of Library. 

Move the last line to Library class